Resources/MessageQueue/MQListener.py

```diff
@@ -0,0 +1,80 @@
+""" MQListener class encapsulates listener functionality fully configurable
+    by the CS options
+
+    The MQListener can be used in two modes:
+
+    Simple mode with a default callback allows to get messages one by one by an explicit
+    get() call:
```

Comment:
You need two colons to turn the indented parts into a nice block
i.e.:
```
get() call::

      listener = MQListener( queueName )
      listener.get()
```
(or prefix with ">>>" or use ".. code-block::" )